src/annotate.rs

```diff
@@ -27,6 +27,7 @@ fn prepare_metadata(wasm_path: PathBuf, metadata_path: PathBuf) -> Result<Metada
         .map_err(|e| anyhow!("Cannot compute ProtocolVersion used by the policy: {:?}", e))?;
 
     metadata.protocol_version = Some(protocol_version);
+    metadata.annotated_by = Some(format!("kwctl v{}", env!("CARGO_PKG_VERSION")));
 
```

Comment:
If we need to process this on a later operation I think it better to write `CARGO_PKG_VERSION`, so we can just use a semver parser directly and do comparisons.

Reply:
resolved as discussed on slack (see below)